internal/generator/generator.go

```diff
@@ -79,6 +79,8 @@ func (kg *KustomizeGenerator) WriteFile(dirPath string) (string, error) {
 		})
 	}
 
+	kus.Components = kg.kustomization.Spec.Components
```

Comment:
What if components are already set in `kustomization.yaml`? Like with `Images` we need to merge the two arrays and add only the items which are not in original config.

Reply:
Please add tests for both use-cases.

Reply:
Oh, I missed that. I have added a test.